### PR TITLE
Fix three failed heavy tests

### PR DESCRIPTION
**Pull request by jiangwen84**
ref #14946

### CODE REVIEW

**Comment by jiangwen84** [comment]
@fdkong

File: modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i [hunk 1]
```diff
@@ -136,6 +136,7 @@
 []
 
 [Bounds]
+  active = ''
```
Comment (fdkong) [inline modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: Does not apply bounds at all?

**Review by fdkong — CHANGES_REQUESTED** [review]

File: modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i (hunk 1 above)
Comment (jiangwen84) [reply modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: It did not use the bounds, see its [Preconditioning] block. 

Our VI solver check will error out if it detects bounds in the input file but does not use VI solver.

Comment (fdkong) [reply modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: Delete these useless blocks?

Comment (fdkong) [reply modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: I prefer to just delete if we do not use at all

Comment (jiangwen84) [reply modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: It is not my code and I try to make minimum changes. 

It is up to you. I can delete them. Let me know.

Comment (fdkong) [reply modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: Yes. Just delete them. Thanks

Comment (jiangwen84) [reply modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i]: Done!

**Review by fdkong — APPROVED** [review]

**Pull request merged**